src/components/common/Extensions/ImageCarousel/ImageCarousel.tsx

```diff
@@ -23,33 +34,59 @@ const ImageCarousel: FC<ImageCarouselProps> = ({
           <div className="flex">
             {slideUrls.map((url) => (
               <div
-                className="min-w-full sm:mr-4 sm:w-[31.875rem] sm:min-w-[31.875rem]"
+                className={
+                  slideWrapperClassName ??
+                  'min-w-full sm:mr-4 sm:w-[31.875rem] sm:min-w-[31.875rem]'
+                }
                 key={url}
               >
                 <img
                   alt="file"
                   src={url}
-                  className="aspect-[380/248] w-full object-cover sm:aspect-[510/248]"
+                  className={
+                    slideImageClassName ??
+                    'aspect-[380/248] w-full object-cover sm:aspect-[510/248]'
+                  }
                 />
               </div>
             ))}
           </div>
         </div>
       </div>
       <div className="absolute bottom-0 m-0 flex w-full justify-center">
-        {scrollSnaps.map((_, index) => (
-          <DotButton
-            // eslint-disable-next-line react/no-array-index-key
-            key={index}
-            onClick={() => scrollTo(index)}
-            className={clsx(
-              'mx-1 h-2 w-2 cursor-pointer rounded-full bg-gray-200 transition-all duration-normal hover:bg-blue-400',
-              {
-                'bg-gray-500': index === selectedIndex,
-              },
-            )}
-          />
-        ))}
+        {scrollSnaps.map((_, index) =>
+          isChangeSlideDotButton ? (
+            <DotButton
+              // eslint-disable-next-line react/no-array-index-key
+              key={index}
+              onClick={() => scrollTo(index)}
+              className={clsx(
+                'mx-1 h-2 w-2 cursor-pointer rounded-full bg-gray-200 transition-all duration-normal hover:bg-blue-400',
+                {
+                  'bg-gray-500': index === selectedIndex,
+                },
+              )}
+            />
+          ) : (
+            <button
+              type="button"
+              aria-label={formatMessage({ id: 'ariaLabel.changeSlide' })}
+              // eslint-disable-next-line react/no-array-index-key
+              key={index}
+              onClick={() => scrollTo(index)}
+              className="group my-[-10px] py-[10px]"
+            >
+              <div
+                className={clsx(
+                  'mx-1 mx-[.3125rem] h-2 h-[.1875rem] w-2 w-[5.875rem] cursor-pointer rounded-full bg-gray-200 transition-all duration-normal group-hover:bg-blue-400',
+                  {
+                    'bg-gray-500': index === selectedIndex,
+                  },
+                )}
+              />
+            </button>
+          ),
+        )}
```

Comment:
I would suggest a very small change here @adam-strzelec if you could please add a constant 
`const isSelectedIndex = index === selectedIndex` and replace the usage of `index === selectedIndex` with `isSelectedIndex`?